Provider documentation fixes (#7823)
traefik/traefik: docs/content/providers/kubernetes-ingress.md

## Enabling and Using the Provider

> The provider can be enabled in the static configuration:

Reviewer: ```suggestion
You can enable the provider in the static configuration:
```

> When using a single instance of Traefik with LetsEncrypt, no issues should be encountered,
> however this could be a single point of failure.
> Unfortunately, it is not possible to run multiple instances of Traefik 2.0 with LetsEncrypt enabled,

Reviewer: ```suggestion
When using a single instance of Traefik Proxy with Let's Encrypt, you should encounter no issues.
However, this could be a single point of failure.
Unfortunately, it is not possible to run multiple instances of Traefik 2.0 with Let's Encrypt enabled,
```

because there is no way to ensure that the correct instance of Traefik receives the challenge request, and subsequent responses.
Previous versions of Traefik used a [KV store](https://doc.traefik.io/traefik/v1.7/configuration/acme/#storage) to attempt to achieve this,
but due to sub-optimal performance that feature was dropped in 2.0.

> If you need LetsEncrypt with HA in a kubernetes environment,

Reviewer: ```suggestion
If you need Let's Encrypt with high availability in a Kubernetes environment,
```

> we recommend using [Traefik Enterprise](https://traefik.io/traefik-enterprise/) where distributed LetsEncrypt is a supported feature.

Reviewer: ```suggestion
we recommend using [Traefik Enterprise](https://traefik.io/traefik-enterprise/) which includes distributed Let's Encrypt as a supported feature.
```

> If you want to keep using Traefik Community Edition,

Reviewer: ```suggestion
If you want to keep using Traefik Proxy,
```
I'm not sure this gets the point across the same way that "community edition" does, but this is the branding we're using now.
